sherpa/astro/tests/test_astro_data_swift_unit.py

```diff
@@ -0,0 +1,454 @@
+#
+#  Copyright (C) 2017  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+"""Test handling of Swift [1]_ data.
+
+At the moment the tests are of routines in the sherpa.astro.io module,
+since the high-level routins like sherpa.astro.ui.unpack_pha are a
+very-thin wrapper around the io routines.
+
+References
+----------
+
+.. [1] The Swift Gamma Ray-Burst Mission https://swift.gsfc.nasa.gov/
+
+"""
+
+import numpy as np
+from numpy.testing import assert_allclose
+
+import pytest
+
+from sherpa.utils import requires_data, requires_fits
+from sherpa.utils.err import IOErr
+from sherpa.astro.data import DataARF, DataPHA, DataRMF
+
+# Should each test import io instead of this? Also, do we have a
+# better way of determining what the backend is?
```

Comment:
The reasons for these comments are - in part - because there's the following comment in `sherpa/astro/io/__init__.py`

> This module should not be imported directly if there is no
FITS support present (i.e. neither the crates or astropy
back ends are in use).

so I'm always a bit nervous about how best to import and use it.